roles/iptables/tasks/iptables.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: iptables variables
+  include_vars: vars/iptables.yml
```

Comment:
This task is probably unnecessary. Try placing the iptables.yml in the role as `roles/iptables/defaults/main.yml`.